File: src/main/java/net/ndrei/teslapoweredthingies/items/BaseThingyItem.java
package net.ndrei.teslapoweredthingies.items;

import net.ndrei.teslacorelib.items.RegisteredItem;
import net.ndrei.teslapoweredthingies.TeslaThingiesMod;

/**
 * Created by CF on 2017-01-07.
 */
public class BaseThingyItem extends RegisteredItem {
    public BaseThingyItem(String registryName) {
        super(TeslaThingiesMod.MODID, TeslaThingiesMod.creativeTab, registryName);
    }
}

File: src/main/kotlin/net/ndrei/teslapoweredthingies/items/BaseThingyItem.kt
package net.ndrei.teslapoweredthingies.items

import net.ndrei.teslacorelib.items.RegisteredItem
import net.ndrei.teslapoweredthingies.TeslaThingiesMod

/**
 * Created by CF on 2017-06-30.
 */
open class BaseThingyItem(registryName: String)
    : RegisteredItem(TeslaThingiesMod.MODID, TeslaThingiesMod.creativeTab, registryName)
